apps/journeys-admin/src/components/JourneyList/JourneyCard/JourneyCardMenu/JourneyCardMenu.stories.tsx

```diff
@@ -1,16 +1,27 @@
 import { Story, Meta } from '@storybook/react'
-
 import { journeysAdminConfig } from '../../../../libs/storybook'
 import JourneyCardMenu from './JourneyCardMenu'
+import { JourneyStatus } from '../../../../../__generated__/globalTypes'
 
-const TestStory = {
+const JoruneyCardMenuDemo = {
```

Comment:
```suggestion
const JourneyCardMenuDemo = {
```
Typo is repeated a couple of times down this page as well